spread_tests/scripts/start_kernos

```diff
@@ -0,0 +1,42 @@
+#! /usr/bin/env bash
+
+set -e
+
+prepare_kernos_runtime()
+{
+    KERNOS_RUNTIME_DIR=$("${SCRIPTS_DIR}/get_kernos_runtime_dir")
+    mkdir -p "${KERNOS_RUNTIME_DIR}"
+    rm -rf "${KERNOS_RUNTIME_DIR:?}/*"
+
+    TASK_DIR="${SPREAD_ROOT_DIR}/${SPREAD_TASK}"
+    LAYERS="${TASK_DIR}/layers"
+
+    cp -r --dereference "${LAYERS}" "${KERNOS_RUNTIME_DIR}/layers"
+    cp -r "${SERVICES_DIR}" "${KERNOS_RUNTIME_DIR}"
+
+    cp "$(which kernos)" "${KERNOS_RUNTIME_DIR}/kernos"
+    cp "$(which busybox)" "${KERNOS_RUNTIME_DIR}/busybox"
+}
+
+ensure_dummy_started()
+{
+    KERNOS_RUNTIME_DIR=$("${SCRIPTS_DIR}/get_kernos_runtime_dir")
+
+    # Spin on this file to ensure the dummy service has been started
+    until [ -e "${KERNOS_RUNTIME_DIR}/dummy_service_started.tok" ]
+    do
+        echo "Waiting for dummy_service_started.tok to exist"
+        sleep 1
+    done
+
+    echo "Found dummy_service_started.tok, preparation finished"
+    sleep 1
+}
+
+
+prepare_kernos_runtime
+
+"${SCRIPTS_DIR}/kernos_daemon" boot-firmware --force &> /dev/null &
```

Comment:
Remove `--force` here and use `KERNOS=1` instead like I suggested above. This way, if we add more checks to the `boot-firmware` command in the future, those will not be skipped.